src/metabase/models/card.clj

```diff
@@ -558,12 +563,6 @@
       maybe-populate-initially-published-at
       (dissoc :id)))
 
-(t2/define-after-update :model/Card
-  [card]
-  (u/prog1 card
-    (when (contains? (t2/changes card) :dataset_query)
```

Comment:
`(t2/changes card)` is not working properly, so this is being called on every update, even `report_card.name`

Reply:
also, see https://github.com/camsaul/toucan2/issues/173

Reply:
Ouch. I have been confused at how much analysis is going on in stats, and this would explain it.